src/targets/browser/index.jsx

```diff
@@ -7,15 +7,21 @@ import CozyClient, { CozyProvider } from 'cozy-client'
 import { render } from 'react-dom'
 import { I18n } from 'cozy-ui/react/I18n'
 import App from 'components/App'
+import { Provider } from 'react-redux'
+import configureStore from 'store/configureStore'
+import { hot } from 'react-hot-loader'
 
 const RootApp = props => (
-  <I18n lang={props.lang} dictRequire={lang => require(`locales/${lang}`)}>
-    <CozyProvider client={props.client}>
-      <App />
-    </CozyProvider>
-  </I18n>
+  <Provider store={props.store}>
+    <I18n lang={props.lang} dictRequire={lang => require(`locales/${lang}`)}>
+      <CozyProvider client={props.client}>
```

Comment:
The `CozyProvider` can also be used to provide the store, which also avoids having 2 separate stores for cozy-client and the rest of the app. You can check out how it's done in drive :)